src/components/ItemList.js

```diff
@@ -1,41 +1,67 @@
-import { collection, query, where, onSnapshot } from 'firebase/firestore';
-import { useEffect, useState } from 'react';
-import { db } from '../lib/firebase.js';
+import { useState } from 'react';
```

Comment:
Nice job cleaning up the imports, in this case by removing `useEffect`, that's no longer needed 👍